src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * DeleteTagCommand removes the tag linked to the task in the task list.
+ */
+public class DeleteTagCommand extends Command {
+    public static final String COMMAND_WORD = "deletetag";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": deletes tags linked to the task.\n"
+            + "Format: " + COMMAND_WORD + " " + "[" + PREFIX_TAG + "TAG]\n"
+            + "Example: " + COMMAND_WORD + " " + "t/" + "priority\n"
+            + COMMAND_WORD + " " + PREFIX_TAG + "priority deadline";
+    public static final String NO_PRIORITY_TAG_TO_DELETE =
+            "There is no priority tag to delete from the task.";
+    public static final String NO_DEADLINE_TAG_TO_DELETE =
+            "There is no deadline tag to delete form the task.";
+    public static final String ALL_TAGS_DELETED_SUCCESSFULLY =
+            "All tags have been deleted from the task.";
```

Comment:
Would it be better to say eg. "Priority tag has been deleted from the task", instead of all, since all suggests removal of both priority and deadline tags rather than only 1 tag.